app/src/main/java/com/example/candyshop/data/db/CandyDatabase.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.candyshop.data.db
+
+import androidx.room.Database
+import androidx.room.RoomDatabase
+
+// provides the app with instances of the DAOs
+@Database(
+    entities = [CandyEntity::class],
+    version = 1
```

Comment:
if you are interested you can also check migration files in our big project. Its not needed here but its always good to have this knowledge why we need migration and when we should use it when working with DB